### PR TITLE
Add cw-band content

### PR DESCRIPTION
**Pull request by RogerKSI**
Details
- add CW-Band contents
- swap Band-VRF and standard dataset section
- add coming soon for integration tools (falcon, PricefeedX)

### CODE REVIEW

**Review by colmazia — APPROVED** [review]

File: docs/integration-tools/pricefeedx/introduction.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<!--
+order: 1
+-->
+
+# PricefeedX (Coming soon)
```
Comment (taobun) [inline docs/integration-tools/pricefeedx/introduction.md]: Remove X

File: docs/integration-tools/cw-band/example.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+# Example Use Cases
+
+As you can request any oracle script in BandChain from your CosmWasm contract. it opens the opportunity for getting any data from the real world such as crypto prices, stock prices, and weather data. Please see the examples of CosmWasm contracts that requests data from BandChain below.
+
+---
+
+### Pricefeed
+
+This price feed contract is an example of getting crypto prices from BandChain.
```
Comment (taobun) [inline docs/integration-tools/cw-band/example.md]: Add more detail like: Using push model to allow other contracts can read the latest price and execute action in one transaction

File: docs/integration-tools/cw-band/example.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+**Requirements:**
+
+- Can trigger function to request new crypto prices from BandChain. (Execute `Request`)
+- Able to query the price of a symbol in the USD unit. (Query `GetRate`)
+- Able to query the price of a symbol in any unit. (Query `GetReferenceData`)
+- Able to bulk query the price of symbols in any unit. (Query `GetReferenceDataBulk`)
+
+You can see the full implementation of the contract [here](https://github.com/bandprotocol/cw-band/tree/main/contracts/price-feed)
```
Comment (taobun) [inline docs/integration-tools/cw-band/example.md:22]: Let's add another example like pull model (request data when action need) and write callback when data has been relayed.

File: docs/integration-tools/cw-band/example.md (hunk 2 above)
Comment (RogerKSI) [reply docs/integration-tools/cw-band/example.md]: added

File: docs/integration-tools/cw-band/example.md (hunk 3 above)
Comment (RogerKSI) [reply docs/integration-tools/cw-band/example.md:22]: added lottery.

File: docs/integration-tools/cw-band/example.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+### Lottery
+
+Lottery dApp is one of the use cases that can do with CW-Band using the pull model to get the winning number when it reaches the end of the period to find the winner.
```
Comment (taobun) [inline docs/integration-tools/cw-band/example.md]: Use "\`cw-band\`" instead

Comment (RogerKSI) [reply docs/integration-tools/cw-band/example.md]: updated.

**Review by taobun — APPROVED** [review]

**Comment by RogerKSI** [comment]
@babybunn This one is ready. please help review and deploy. :pray:

**Review by satawatnack — APPROVED** [review]

**Pull request merged**